### PR TITLE
feat: implement an additional experimental syncer, the `ossf/scorecard` project

### PR DESCRIPTION
**Pull request by patrickdevivo**
more context can be found here: https://github.com/ossf/scorecard. Produces a score for a repo, stores the results as JSON in the DB, and creates some views for easier consumption of the data.

### CODE REVIEW

File: migrations/900000000000021_ossf_scorecard.up.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
+BEGIN;
+
+INSERT INTO mergestat.repo_sync_types (type, description, short_name, priority, type_group)
+VALUES ('OSSF_SCORECARD_REPO_SCAN', 'Executes an OSSF scorecard scan on a git repository', 'OSSF Scorecard Repo Scan', 3, 'GITHUB') ON CONFLICT DO NOTHING;
+
+CREATE TABLE IF NOT EXISTS ossf_scorecard_repo_scans (repo_id uuid PRIMARY KEY, results jsonb NOT NULL);
```
[the remark this answers is not in the document]
Comment (patrickdevivo) [reply migrations/900000000000021_ossf_scorecard.up.sql]: good call!

**Review by amenowanna — DISMISSED** [review]

**Review by amenowanna — APPROVED** [review]

File: migrations/900000000000021_ossf_scorecard.up.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
+BEGIN;
+
+INSERT INTO mergestat.repo_sync_types (type, description, short_name, priority, type_group)
+VALUES ('OSSF_SCORECARD_REPO_SCAN', 'Executes an OSSF scorecard scan on a git repository', 'OSSF Scorecard Repo Scan', 3, 'GITHUB') ON CONFLICT DO NOTHING;
```
Comment (riyaz-ali) [inline migrations/900000000000021_ossf_scorecard.up.sql:4]: Why is this made part of `GITHUB` `type_group`?

Comment (amenowanna) [reply migrations/900000000000021_ossf_scorecard.up.sql:4]: You will see in the sync file this uses the GitHub credentials. We need it to share the same concurrency for that PAT. We were able to use the group here to account for that. We still need to take that into consideration for auto imports.

Comment (riyaz-ali) [reply migrations/900000000000021_ossf_scorecard.up.sql:4]: 👍

**Pull request merged**